### ISSUE TITLE
About the label-only setting

### ISSUE THREAD

**Issue by cmhcbb** [labsix/limited-blackbox-attacks#5]
Hi, 
   I find the loss is always zero when running the label-only.sh. However, I could get a result in partial information and query-limited setting. How could I solve the problem?

**Comment by andrewilyas** [comment]
Hello! Are you running the command on the same data (imageNet validation set)?

Two things to keep in mind are:
(a) The attack needs to run much longer than the partial-info and query-limited attacks, so make sure you are letting it run for sufficiently long
(b) If it's a different dataset, sometimes the query radius parameter needs some tuning.

**Comment by andrewilyas** [comment]
Sorry actually, reading this again, could you clarify what "zero loss" means?

Since these are targeted adversarial attacks, zero loss is typically a good thing.

**Comment by cmhcbb** [comment]
The loss is zero in the first step, which doesn't make sense. I have tried several images and several targets and the loss is zero every time.

**Comment by cmhcbb** [comment]
I am using ImageNet validation for all three experiments and I just use the default hyper-parameter.

**Comment by andrewilyas** [comment]
Try running the attack for longer.
 
I will look into this more later, but intuitively it seems as though the loss should start at 0, because we start at an image of the target class. As the epsilon is decreased, the loss should start to fluctuate more.

**Comment by cmhcbb** [comment]
Any updates?

**Comment by andrewilyas** [comment]
Sorry! This was addressed in #3 ; will be fixed soon!

**Comment by andrewilyas** [comment]
This has been fixed.
